ingest-app/src/cmr/ingest/api/subscriptions.clj

```diff
@@ -165,10 +189,12 @@
                             content-type
                             headers)
           native-id (get-unique-native-id request-context tmp-subscription)
-          new-subscription (assoc tmp-subscription :native-id native-id)
-          subscriber-id (get-subscriber-id new-subscription)]
+          subscriber-id (get-subscriber-id tmp-subscription)
+          token-user (api-core/get-user-id-from-token request-context)
```

Comment:
`token-user` is not used in this function, move this down to `add-id-and-email-to-metadata-if-missing`.

Reply:
Agreed - this has been changed.